src/js/versions/version_controller.js

```diff
@@ -77,19 +77,27 @@ export class VersionController extends Component {
         this._versions = value;
     }
 
+    get selectedLevel() {
```

Comment:
`selectedLevel` is not descriptive - something like `selectedGeoType` would be better. But `controller.state.preferredChild` stores the same data. do we need this field?

Reply:
Will change it to selectedGeoType 
We need to it for updating and selecting accurate level when a version is selected and have multiple values
https://github.com/OpenUpSA/wazimap-ng-ui/pull/625/files#diff-5419bcf277caec6d42c05c30a4adc5b4406c900192b319b8d2bf6df0530032afR31